horace_core/sqw/file_io/@faccess_sqw_v2/private/upgrade_file_format_.m

```diff
@@ -53,15 +53,18 @@
 head.u_to_rlu = zeros(4,4);
 head.ulen = ones(1,4);
 head.ulabel = {'a','b','c','d'};
-head.instrument = struct();
-head.sample = struct();
+head.instruments = struct();
+head.samples = struct();
+head.alatt = [1,1,1];
+head.angdeg = [90,90,90];
 if nfiles>1
     heads = cell(1,nfiles);
     % matlab bug fixed in 2016b
     heads  = cellfun(@(x)gen_head(head,x),heads,'UniformOutput',false);
 else
     heads = head;
 end
+heads = Experiment(heads);
```

Comment:
well, there is header written in old file format and it will remain header. So I am hesitant about this change -- this will bring confusion. 
Better to add set/get experiment to the interface and work with this (but this has been already discussed)